<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="QOpenTLD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QOpenTLD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/rcs.qrc">
    <normaloff>:/res/img/opentld.png</normaloff>:/res/img/opentld.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_general">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_general">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>384</width>
            <height>810</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_loadModel">
             <property name="title">
              <string>Load Model</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_modelPath">
                <property name="text">
                 <string>Model</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_modelPath"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_modelPath">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_initialBoundingBox">
             <property name="title">
              <string>Initial Bounding Box</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_initialBoundingBoxX">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_initialBoundingBoxY">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_initialBoundingBoxW">
                <property name="text">
                 <string>w</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_initialBoundingBoxH">
                <property name="text">
                 <string>h</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_initialBoundingBoxX">
                <property name="text">
                 <string>100</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_initialBoundingBoxY">
                <property name="text">
                 <string>100</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_initialBoundingBoxW">
                <property name="text">
                 <string>100</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_initialBoundingBoxH">
                <property name="text">
                 <string>100</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_theta">
             <property name="text">
              <string>Theta</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="lineEdit_theta">
             <property name="text">
              <string>0.8</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_trajectory">
             <property name="title">
              <string>Trajectory</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_trajectory">
                <property name="text">
                 <string>Trajectory Frames</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_trajectory">
                <property name="text">
                 <string>20</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_learning">
             <property name="text">
              <string>Learning</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_selectManually">
             <property name="text">
              <string>Select Manually</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_showOutput">
             <property name="text">
              <string>Show Output</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_showNotConfident">
             <property name="text">
              <string>Show Not Confident</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_showColorImage">
             <property name="text">
              <string>Show Color Image</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_showDetections">
             <property name="text">
              <string>Show Detections</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_showForeground">
             <property name="text">
              <string>Show Foreground</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_saveOutput">
             <property name="title">
              <string>Save Output</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_outputDir">
                <property name="text">
                 <string>Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_outputDir"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_outputDir">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_printResults">
             <property name="title">
              <string>Print Results</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_printResults">
                <property name="text">
                 <string>File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_printResults"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_printResults">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_printTiming">
             <property name="title">
              <string>Print Timing</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_printTiming">
                <property name="text">
                 <string>File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_printTiming"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_printTiming">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_alternating">
             <property name="text">
              <string>Alternating</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_exportModel">
             <property name="text">
              <string>Export Model</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_aquisition">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Aquisition</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QScrollArea" name="scrollArea_aquisition">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_aquisition">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>400</width>
            <height>299</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_imagePath">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_method">
             <property name="text">
              <string>Method</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_imagePath">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>File</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_method">
             <item>
              <property name="text">
               <string>Camera</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Images</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Video</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_imagePath">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QGroupBox" name="groupBox_lastFrame">
             <property name="title">
              <string>Last Frame</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_lastFrame">
                <property name="text">
                 <string>Last Frame</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_lastFrame">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_startFrame">
             <property name="text">
              <string>Start Frame</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_startFrame">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_detector">
      <attribute name="title">
       <string>Detector</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QScrollArea" name="scrollArea_detector">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_detector">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>400</width>
            <height>299</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_proportionalShift">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Proportional Shift</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_proportionalShift">
                <property name="text">
                 <string>Proportional Shift</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_proportionalShift">
                <property name="inputMethodHints">
                 <set>Qt::ImhNone</set>
                </property>
                <property name="inputMask">
                 <string/>
                </property>
                <property name="text">
                 <string>0.1</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_minScale">
             <property name="text">
              <string>Min Scale</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_minScale">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_maxScale">
             <property name="text">
              <string>Max Scale</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_maxScale">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_numberOfFeatures">
             <property name="text">
              <string>Number of Features</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_numberOfFeatures">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_numberOfTrees">
             <property name="text">
              <string>Number of Trees</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_numberOfTrees">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_minSize">
             <property name="text">
              <string>Min Size</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_minSize">
             <property name="text">
              <string>25</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_modelPath</tabstop>
  <tabstop>lineEdit_proportionalShift</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>lineEdit_modelPath</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/rcs.qrc"/>
 </resources>
 <connections/>
</ui>
